scss/_orange-navbar.scss

```diff
@@ -61,3 +48,34 @@
     }
   }
 }
+
+.header-minimized {
+  @include media-breakpoint-up(md){
+    padding-top: map-get($spacers, 2);
+    padding-bottom: 0;
+    .nav-item:last-of-type {
+      .nav-icon {
+        padding-right: 0 !important;
```

Comment:
If possible get rid of this `!important`, if not fix:
```
 58:27  ✖  Unexpected !important             declaration-no-important
```

Reply:
For the moment, special comment to disable important check has been put